docs/020-deployment/howtos/020-setup-sso.md

```diff
@@ -72,22 +72,41 @@ For more information about the client secret, see [Microsoft documentation](http
 
 Redirect URI is the location to which the Microsoft identity platform redirects the user's client and sends security tokens after authentication.
 
+In the setup below, we will be working with fictitious domain 'yourdomain.com' and the two subdomains 'app' and 'departmentX'. 
+
+**departmentX** references the main CluedIn UI you would access.  
+And **app** references the backend application. By default, it is just app, but can be changed in the values file.
+
```

Comment:
```suggestion
In the setup below, we will be working with fictitious domain 'yourdomain.com' and the two subdomains:

- 'app' – references the main CluedIn UI you would access.

 - 'departmentX' – references the back-end application. By default, it is just app, but can be changed in the values file.

```

Reply:
I have amended this as there was a leading space, and i think bullet points are find to be together

Reply:
@SolomiiaSnihur are you happy with amendment above?